packages/client/python-apypie/python-apypie.spec

```diff
@@ -0,0 +1,65 @@
+# Created by pyp2rpm-3.3.2
+%global pypi_name apypie
+
+Name:           python-%{pypi_name}
+Version:        0.0.3
+Release:        1%{?dist}
+Summary:        Apipie bindings for Python
+
+License:        MIT
+URL:            https://github.com/Apipie/apypie
+Source0:        https://files.pythonhosted.org/packages/source/a/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python2-devel
+BuildRequires:  python2-setuptools
+
+BuildRequires:  python%{python3_pkgversion}-devel
```

Comment:
Contrary to EPEL7, our builders don't have Python3 support. So I'll have to disable that.